crates/integration_test/src/tests/lifecycle/state.rs

```diff
@@ -1,18 +1,10 @@
-use crate::utils::get_runtime_path;
+use crate::utils::test_utils::get_state;
 use std::io;
 use std::path::Path;
-use std::process::{Command, Stdio};
 use test_framework::TestResult;
 
 pub fn state(project_path: &Path, id: &str) -> TestResult {
-    let res = Command::new(get_runtime_path())
-        .stdout(Stdio::piped())
-        .stderr(Stdio::piped())
-        .arg("--root")
-        .arg(project_path.join("runtime"))
-        .arg("state")
-        .arg(id)
-        .spawn()
+    let res = get_state(id, project_path)
```

Comment:
Given that you have added the function get_state_output, can we use it directly here? As when the state command is successful, we are converting the stdout and stderr to string, both here and in the get_state_output fn. So can we just call that and the work on the result returned by it?